source/qb64pe.bas

```diff
@@ -26063,6 +26077,40 @@ SUB increaseUDTArrays
     REDIM _PRESERVE udtenext(x + 1000) AS LONG
 END SUB
 
+FUNCTION CompareVersions (v1$, v2$)
+    DIM AS LONG p1, p2, p3, p4, p5, p6 'part 1 through 6
+
+    IF GetVersionValues(v1$, p1, p2, p3) = 0 THEN CompareVersions = 666: EXIT FUNCTION 'Evil Error code.  Something went wrong!
+    IF GetVersionValues(v2$, p4, p5, p6) = 0 THEN CompareVersions = 666: EXIT FUNCTION 'Evil Error code.  Something went wrong!
```

Comment:
I think this should produce an actual compilation error, that way people can't hit it accidentally. It's not super easy for them to test the version comparisons, so they might think that Ex. `$IF VERSION < 3.5 THEN` is working even though it's actually always treated as false.